castle/cms/tiles/querylisting.py

```diff
@@ -269,7 +269,16 @@ def filter_pattern_config(self):
             'url': self.this_url,
             'selector': '#query-results-%s' % self.id or ''
         }
-        return json.dumps(config)
+        
+        out = '{}'
+        try:
+            out = json.dumps(config)
+        except UnicodeError:
+            try:
+                out = json.dumps(config, ensure_ascii=False)
+            except UnicodeError:
+                return '{}'
```

Comment:
This is nitpicking, but this line could just set out to `{}` (or just `pass` maybe) and you'd only have one exit point. That's my only feedback.